packages/portal/pages/games/[game_id].js

```diff
@@ -0,0 +1,20 @@
+import PouchDB from 'pouchdb'
+import React, { useEffect, useMemo, useState } from 'react'
+
+function Game({ game_id }) {
+  const [info, setInfo] = useState('')
+
+  useEffect(() => {
+    const db = new PouchDB(`/api/games/${game_id}`, {
```

Comment:
How does the PouchDB instance know where the CouchDB instance is running? Is this information coming from the info below?